src/releng_docs/services_shipit.rst

```diff
@@ -3,53 +3,25 @@
 Service family: ``src/shipit_*``
```

Comment:
It looks like the only thing changing here is formatting? Please correct me if I'm wrong, this diff is really hard to read.